code/workspaces/web-app/src/api/messagesService.js

```diff
@@ -0,0 +1,64 @@
+import { gqlMutation, gqlQuery } from './graphqlClient';
+import errorHandler from './graphqlErrorHandler';
+
+function createMessage(message) {
+  const mutation = `
+    CreateMessage($cluster: ClusterCreationRequest!) {
+      createMessage(cluster: $cluster) {
+        id
+        message
+        expiry
+        created
+      }
+    }
+  `;
+
+  return gqlMutation(mutation, { message })
```

Comment:
Can we shift away from promise chaining here and use async/await syntax - it'll make it easier to add other operations or introduce some specific error handling here.

Reply:
Good point, updated to use async/await now